requirements/edx/testing.txt

```diff
@@ -1141,9 +1136,11 @@ random2==1.0.1
     # via -r requirements/edx/base.txt
 recommender-xblock==2.0.1
     # via -r requirements/edx/base.txt
-redis==4.1.1
-    # via -r requirements/edx/base.txt
-regex==2021.11.10
+redis==3.5.3
```

Comment:
not sure why `redis` is being downgraded if it has been working fine we shouldn't downgrade it